app/jobs/create_exif_technical_metadata_job.rb

```diff
@@ -0,0 +1,59 @@
+class CreateExifTechnicalMetadataJob < ApplicationJob
+  def perform(file_set, file_path)
+    exif_data = get_exif_data(file_path)
+    populate_tech_md(exif_data, file_set)
+  end
+
+  private
+
+    # Calls the Vendored Exif Tool with appriorate arguments and returns the hash
+    def get_exif_data(file_path)
+      Exiftool.new(file_path, '-a -u -g1').to_hash
+    end
+
+    def populate_tech_md(exif_data, fs)
+      made_new_tech_md = false
+      t = fs.technical_metadata
+      if t.nil?
+        t = TechnicalMetadata.new
+        made_new_tech_md = true
+      end
+      t = populate_fields(t, exif_data, fs.id)
+      fs.members << t if made_new_tech_md # if we made a new techMD add it as a member
+      fs.update_index
+    end
+
+    # rubocop:disable Metrics/AbcSize
```

Comment:
This seems like an awful lot of disabling. I totally understand disabling rubocop when we copy in overrides for Hyrax files but for our own new stuff, shouldn't we try to respect rubocop?